admin_guide/aggregate_logging.adoc

```diff
@@ -233,7 +233,7 @@ Enabling aggregated logging to *Elasticsearch* involves:
 
 . link:#creating-an-elasticsearch-cluster[Creating an *Elasticsearch* cluster]
 . link:#creating-logging-pods[Creating logging pods]
-. link:#creating-the-kibana-service[Creating the *Kibana* service]
+. link:#creating-the-kibana-search[Creating the *Kibana* service]
```

Comment:
Don't mind me if it was done like this on purpose for some reason, but from a glance I think the typo fix should have been the other way around. That is, leaving it `#creating-the-kibana-service` here and correcting the typo'd heading anchor `[[creating-the-kibana-search]]` later in this topic, since the heading title is actually "Creating the Kibana Service".